draft-ietf-webtrans-http2.md

```diff
@@ -376,16 +376,13 @@ The WT_STOP_SENDING frame defines the following fields:
 
 ## WT_STREAM Frames
 
-WT_STREAM frames implicitly create a stream and carry stream data. The Type
-field in the WT_STREAM frame takes the form 0b00001XXX (or the set of values
-from 0x08 to 0x0f) to maximize compatibility with QUIC. However, unlike QUIC,
-there are only one bit used to determine the fields that are present in the
-frame:
+WT_STREAM frames carry stream data.
```

Comment:
Asides from whether or not this really needs its own paragraph, do we somewhere want to keep that a stream frame creates a stream? (no preference around wording/implicitness/that stuff, but we no longer say in this section at all that it creates a stream)

Reply:
The definition starts with "WT_STREAM frames implicitly create a stream [...]"  I figured that would be OK.

Reply:
Sorry, I must just be misreading the diff. It looks to me like 
> implicitly create a stream

is no longer in the document anywhere, which is fine from a wording perspective, but we probably want _some_ mention that they create a stream.

Reply:
Put create a stream back in that sentence in [60b744e](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/45/commits/60b744e1b8e925f2a87550d895192c4d8023190f), but we can always reword/take it out later if needed.